docs/tutorials/medical/download_and_split_ixi.py

```diff
@@ -94,7 +94,7 @@ def download_and_extract_ixi_sample(root_folder):
         cfg_file = os.path.join(cfg_folder, f'{center_name.lower()}.ini')
 
         print(f'Creating node at: {cfg_file}')
-        config = NodeConfig(root=environ["ROOT_DIR"], name=f'{center_name.lower()}.ini', auto_generate=False)
+        config = NodeConfig(root=config.root, name=f'{center_name.lower()}.ini', auto_generate=False)
```

Comment:
`config.root` is not existant there.
Do we mean ?

```
config = NodeConfig(root=root_folder, name=f'{center_name.lower()}.ini', auto_generate=False)
```

Apply same modification to `notebooks/medicalimage-segmentation/medical-image-segmentation-custom-unet-code.ipynb`

Reply:
Thanks!  yes it should be `root_folder`

Reply:
Thank you !

>  I removed this line form the python script download_and_split_ixi.py but I couldn't find it in notebooks/medicalimage-segmentation/medical-image-segmentation-custom-unet-code.ipynb. Do I miss something?

Sorry, my bad, forget it